pkg/tests/ossm/bug_SMM_delete_test.go

```diff
@@ -0,0 +1,67 @@
+package ossm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+// referance - https://issues.redhat.com/browse/OSSM-2374
+
+// testing Jira - https://issues.redhat.com/browse/OSSM-3450
+
+func TestSMMDelete(t *testing.T) {
+	NewTest(t).Id("T39").Groups(Full).Run(func(t TestHelper) {
+		t.Log("Verify the SMMR will be creates Automatically with SMM and SMMR is not deleted when SMM is deleted")
+		t.Cleanup(func() {
+			oc.DeleteNamespace(t, "my-awesome-project")
+			oc.ApplyString(t, meshNamespace, smmr) // If we apply the default SMMR over the current will be updated
+		})
+
+		t.LogStep("Delete default SMMR in mesh namespace and create new ServiceMeshMember in namespace: bookinfo and my-awesome-project. Expected: SMMR will be created automatically")
+		shell.Execute(t, fmt.Sprintf(`oc delete smmr -n %s default`, meshNamespace))
+		oc.CreateNamespace(t, "bookinfo")
+		oc.CreateNamespace(t, "my-awesome-project")
```

Comment:
Let's use one of the standard namespaces that the other tests use (foo, bar, legacy, or mesh-external)